commands/general_commands.galaxy

```diff
@@ -2024,4 +2024,152 @@ void libapx_define_helpmessages(string command_name) {
 		g_commands_guide_entry_general, 
 		command_name, 
 		"This command toggles help messages. <br> _helpmessages on/off <br> Examples: <br> _helpmessages off");
-}
\ No newline at end of file
+}
+// libapx_command_dataedit
+//  ===================
+// changes field values in data catalog entries
+void libapx_command_dataedit(int user, unitgroup u, string argv, int spawningCount, string bracketText) {
+    string msg1= StringWord(argv,3);
+    string msg2= StringWord(argv,4);
+    string msg3= StringWord(argv,5);
+	unitgroup g = libapx_player_parser[user][spawningCount].context;
+//Because CatalogFieldValueSet was being uncooperative with strings, I had to improvise with repetitive lists.

```

Comment:
please, IDENT YOUR FUCKING LINES PSYCHO (kidding, but please indent your shit, it hurts my eyes. Ctrl + [ to move the selected text one indent left, vice versa for Ctrl + ])

Reply:
Oh thanks!